var/spack/repos/builtin/packages/py-mosek/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import platform
+import sys
+
+from spack.package import *
+
+
+class PyMosek(PythonPackage):
+    """MOSEK is a package for large-scale convex and mixed-integer optimization, including LP, QP, SOCP, exponential and power cone problems, SDP, QCQP and MIP. Documentation and examples can be found at the https://www.mosek.com/documentation. MOSEK requires a license. Free of charge trial licenses as well as licenses for non-commercial academic use can be obtained from https://www.mosek.com."""
+
+    homepage = "https://www.mosek.com"
+
+    url = "https://pypi.org/project/Mosek/"
+
+    maintainers("bourdin")
+
+    license("https://mosek.com/products/license-agreement", checked_by="bourdin")
+
+    arch = platform.machine()
+    os = sys.platform
+
+    if os == "darwin":
+        version(
+            "10.2.8",
+            url="https://files.pythonhosted.org/packages/c8/e8/277d457303360f7e9a226915b98df8537fcf9565d9d86c7e4dc2edb8f795/Mosek-10.2.8-cp38-abi3-macosx_11_0_arm64.whl",
```

Comment:
For easier version updates, use the url without hash:
```suggestion
            url="https://files.pythonhosted.org/packages/cp38/m/mosek/Mosek-10.2.8-cp38-abi3-macosx_11_0_arm64.whl",
```
Similar below.